package.json

```diff
@@ -19,17 +19,20 @@
     "@okta/okta-signin-widget": "^3.4.2",
     "@urql/exchange-graphcache": "^2.0.0",
     "axios": "^0.19.0",
+    "bootstrap": "^4.4.1",
```

Comment:
Why is bootstrap being used? Is it only being used for tests?

Reply:
It was the easiest and fastest way to set up the modals they are currently using. They plan to refactor it but needed something up quickly so they could get people working on the rest of the feature.

Reply:
Sounds good!